.github/SECURITY.md

```diff
@@ -0,0 +1,125 @@
+# Oppia Vulnerability Disclosure Process
```

Comment:
This file should be in repository root, not in `.github`

Reply:
From the [github docs](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository):

> To give people instructions for reporting security vulnerabilities in your project, you can add a *SECURITY.md* file to your repository's root, `docs`, or `.github` folder.

Reply:
No worries. We usually prefer to keep these files in `.github` just so the root of the repository doesn't get too cluttered